File: mihabitat-web/src/main/webapp/recursos/js/app/reportes/movimiento.js
var Movimiento = function() {
    
    var self = this;
    
    self.cuenta = new Cuenta();
    self.debe = ko.observable();
    self.descripcion = ko.observable();
    self.fecha = ko.observable();
    self.haber = ko.observable();
    
    self.nombre = ko.observable();
    self.referencia = ko.observable();    
    
    self.limpiar = function() {
        self.cuenta.limpiar();
        self.debe(undefined);
        self.descripcion(undefined);
        self.fecha(undefined);
        self.haber(undefined);
        
        self.nombre(undefined);
        self.referencia(undefined);
    }
    
    self.cargar = function(data) {
        self.cuenta.cargar(data ? data.cuenta : undefined);
        self.debe(data ? data.debe : undefined);
        self.descripcion(data ? data.descripcion : undefined);
        self.fecha(data ? data.fecha : undefined);
        self.haber(data ? data.haber : undefined);
        
        self.nombre(data ? data.nombre : undefined);
        self.referencia(data ? data.referencia : undefined);
    }
}

File: mihabitat-web/src/main/java/com/bstmexico/mihabitat/web/dto/reportes/Movimiento.java
package com.bstmexico.mihabitat.web.dto.reportes;

import java.math.BigDecimal;
import java.util.Date;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonFormat.Shape;

/**
 * @author Pablo Cruz Santos
 * @version 1.0
 * @since 2015
 */
public class Movimiento {

	private Cuenta cuenta;

	private BigDecimal debe;

	private String descripcion;

	@JsonFormat(shape = Shape.STRING, pattern = "dd-MM-yyyy", timezone = "America/Mexico_City")
	private Date fecha;

	private BigDecimal haber;

	/**
	 * Propiedad reporte cajas y bancos
	 */
	private String referencia;

	/**
	 * Propiedad reporte cajas y bancos
	 */
	private String nombre;

	public Movimiento() {
		super();
	}

	public Cuenta getCuenta() {
		return cuenta;
	}

	public void setCuenta(Cuenta cuenta) {
		this.cuenta = cuenta;
	}

	public BigDecimal getDebe() {
		return debe;
	}

	public void setDebe(BigDecimal debe) {
		this.debe = debe;
	}

	public String getDescripcion() {
		return descripcion;
	}

	public void setDescripcion(String descripcion) {
		this.descripcion = descripcion;
	}

	public Date getFecha() {
		return fecha;
	}

	public void setFecha(Date fecha) {
		this.fecha = fecha;
	}

	public BigDecimal getHaber() {
		return haber;
	}

	public void setHaber(BigDecimal haber) {
		this.haber = haber;
	}

	public String getReferencia() {
		return referencia;
	}

	public void setReferencia(String referencia) {
		this.referencia = referencia;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
}
